pycbc/events/events.py

```diff
@@ -337,7 +336,10 @@ def cluster_template_events(self, tcolumn, column, window_size):
         """
         cvec = self.template_events[column]
         tvec = self.template_events[tcolumn]
-        indices = findchirp_cluster_over_window(tvec, cvec, window_size)
+        if window_size == 0:
+            indices = numpy.arange(len(tvec))
```

Comment:
This change would probably mean that you don't need the change in `findchirp_cluster` above.

Reply:
Didn't see this before commenting on the previous thing. I agree :p